ui/src/shared/components/DragAndDrop.tsx

```diff
@@ -146,11 +146,14 @@ class DragAndDrop extends PureComponent<Props, State> {
 
     return (
       <span className="drag-and-drop--buttons">
-        <button className="btn btn-sm btn-success" onClick={this.handleSubmit}>
+        <button
+          className="button button-sm button-success"
+          onClick={this.handleSubmit}
+        >
           {submitText}
         </button>
         <button
-          className="btn btn-sm btn-default"
+          className="button button-sm button-default"
```

Comment:
Want to import the clockface button and use it here? That way we don't have to worry about this component falling out of sync